site/content/docs/howto/configure-supervisor-with-gitlab.md

```diff
@@ -101,14 +82,60 @@ stringData:
   clientSecret: "<your-client-secret>"
 ```
 
-To validate your configuration, run:
+Once your OIDCIdentityProvider has been created, you can validate your configuration by running:
 
 ```shell
-kubectl describe OIDCIdentityProvider gitlab
+kubectl describe OIDCIdentityProvider -n pinniped-supervisor gitlab
 ```
 
 Look at the `status` field. If it was configured correctly, you should see `phase: Ready`.
 
+### (Optional) Use a different GitLab claim for Kubernetes usernames
+
+You can also use other GitLab claims as the username.
+To do this, make sure you have configured the appropriate scopes on your GitLab application, such as `email`.
+
+You must also adjust the `spec.authorizationConfig` to request those scopes at login and adjust `spec.claims` to use those claims in Kubernetes, for example:
+
+```yaml
+# [...]
+spec:
+  # Request any scopes other than "openid" that you selected when
+  # creating your GitLab application. The "openid" scope is always
+  # included.
+  #
+  # See here for a full list of available claims:
+  # https://docs.gitlab.com/ee/integration/openid_connect_provider.html
+  authorizationConfig:
+    additionalScopes: [ email ]
+  claims:
+    username: email
+    groups: groups
+# [...]
+```
+
+### (Optional) Use a private GitLab instance
+
+To use privately hosted instance of GitLab, you can change the `spec.issuer` and `spec.tls.certificateAuthorityData` fields, for example:
+
+```yaml
+apiVersion: idp.supervisor.pinniped.dev/v1alpha1
+kind: OIDCIdentityProvider
+# [...]
+spec:
+  # Specify your GitLab instance URL.
+  issuer: https://gitlab.your-company.example.com.
+
+  # Specify the CA bundle for the GitLab server as base64-encoded PEM
+  # data. This will only be needed for self-managed GitLab. For example,
+  # the output of `cat my-ca-bundle.pem | base64`.
+  # 
+  # This configuration is only necessary if your instance uses a custom CA.
```

Comment:
`This will only be needed for self-managed GitLab.`
`This configuration is only necessary if your instance uses a custom CA.`
Can these two sentences be condensed to something like `This will only be needed if your self-managed GitLab instance uses a custom CA`?